modules/acquia_cms_common/acquia_cms_common.module

```diff
@@ -381,6 +381,19 @@ function acquia_cms_common_modules_installed($modules, $is_syncing) {
       // Set telemetry to be TRUE on install of acquia_connector.
       \Drupal::state()->set('acquia_connector.telemetry.opted', TRUE);
     }
+
+    // Add entity_clone permission after module install.
+    if (in_array('entity_clone', $modules)) {
+      $roles = [
+        'content_administrator',
+        'content_author',
+        'content_editor',
+      ];
+      $permissionFacade = Drupal::classResolver(PermissionFacade::class);
+      foreach ($roles as $role) {
```

Comment:
Within the same method this logic is repeated

Reply:
Done